packages/docusaurus-theme-classic/src/theme/BlogAuthorPage/AuthorsListByLetter/index.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {
+  listAuthorsByLetters,
+  type AuthorLetterEntry,
+} from '@docusaurus/theme-common';
+import Author from '@theme/BlogAuthorPage/Author';
+import type {Props} from '@theme/BlogAuthorPage/AuthorsListByLetter';
+import Heading from '@theme/Heading';
+import styles from './styles.module.css';
+
+function AuthorLetterEntryItem({
+  letterEntry,
+}: {
+  letterEntry: AuthorLetterEntry;
+}) {
+  return (
+    <article>
+      <Heading as="h2" id={letterEntry.letter}>
+        {letterEntry.letter}
+      </Heading>
+      <ul className="padding--none">
+        {letterEntry.authors.map((author) => (
+          <li key={author.permalink} className={styles.author}>
+            <Author {...author} />
+          </li>
+        ))}
+      </ul>
+      <hr />
+    </article>
+  );
+}
+
+export default function AuthorsListByLetter({authors}: Props): JSX.Element {
+  const letterList = listAuthorsByLetters(authors);
```

Comment:
> I wonder if we should group authors by letters.

It would look like this:

- https://cronica-it.github.io/evenimente/autori/

> To me it's unlikely authors will grow as fast as tags.

In an historical site like mine, it is highly likely that each new contribution will come from a new author.

> how do we handle authors that have no name

Good question, it gave me quite some headaches; in the end I added a new object, NamedAuthors.

Reply:
@ilg-ul we can't take your site as an example for this feature because we are building a blog plugin, not a chronological site. We enable you to render the list the way you want with swizzling, but it doesn't mean it's what we should provide by default for people that are actually building blogs.

However I'm fine with the current UI, it's not awesome but good enough to ship:
https://deploy-preview-10216--docusaurus-2.netlify.app/blog/authors

![CleanShot 2024-06-21 at 17 55 58](https://github.com/facebook/docusaurus/assets/749374/de774511-b140-44d8-8392-9ee2bf5c57b9)

Important to note that we only list authors on this page coming from `authors.yml`. Inline authors won't appear there.

Reply:
The look is not relevant; the functionality seems equivalent (even better), so no problems so far.

Also no problems to ignore inline authors.